test/test_pkg_test.py

```diff
@@ -40,16 +58,24 @@ def _build_pkg():
 
 def test_pkg_test():
     """
-    Running a mulled-build test shouldn't cause any errors
+    Running a mulled-build test shouldn't cause any errors.
     """
-    built_package = _build_pkg()
+    built_package = _build_pkg(RECIPE_ONE)
     res = pkg_test.test_package(built_package)
 
 
 def test_pkg_test_mulled_build_error():
     """
-    Make sure calling mulled-build with the wrong arg fails correctly
+    Make sure calling mulled-build with the wrong arg fails correctly.
     """
-    built_package = _build_pkg()
+    built_package = _build_pkg(RECIPE_ONE)
     with pytest.raises(sp.CalledProcessError):
         res = pkg_test.test_package(built_package, mulled_args='--wrong-arg')
+
+
+def test_pkg_test_custom_base_image():
```

Comment:
@johanneskoester nearly working. OSX is still broken I guess we need to skip the tests on OSX with `@pytest.fixture(scope='module', params=PARAMS, ids=IDS)`. I just don't understand why this was passing before.